### ISSUE TITLE
Screen Reader focus is not moving with the keyboard focus on the Formula suggestion Dropdown list

### ISSUE THREAD

**Issue by v-raipra** [nvaccess/nvda#7966]
User Experience:
A user who has vision issues will not be able to navigate to the correct option which he wants to select as NVDA is not reading the content under dropdown.
 
Test Environment:
Build: 17.7806.1000
OS: Win 10 RS PRERELEASE 15008.1000
Tool used: NVDA

 Steps:
1. Open Microsoft Excel.
2. To create a new workbook, navigate to "Create" button on the Excel dialog box using keyboard.
3. Select any cell using keyboard.
4. Press "=" (equal key) and start typing formula in order to populate the entries of the Autocomplete in the form of dropdown.
5. Observe the issue.
 
Actual:
While navigating inside formula suggestion it only announce first of formula even while navigating on other suggestion it still announce the first one and it should announce that list of formula has been open as suggestion
 
Expected:
NVDA should announce the item on which currently focus is instead of announcing the first one and it should also announce that suggestion list is there.

Note:
issue not reproes with Narrator and Jaws.
[New Compressed (zipped) Folder.zip](https://github.com/nvaccess/nvda/files/1702890/New.Compressed.zipped.Folder.zip)
